Overview.src.html

```diff
@@ -2087,6 +2094,11 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
      "<code title>manual</code>" and <var title>response</var>'s
      <span title=concept-response-type>type</span> is "<code title>opaqueredirect</code>".
     </ul>
+
+   <li>
+     <p>Execute <a href=https://w3c.github.io/webappsec-csp/set-response-policy-list>Set
+     <var title>response</var>'s policy list</a> on <var title>response</var>.
+     <span data-anolis-ref>CSP</span>
```

Comment:
Sorry, I don't follow the question.

This bit takes care of responses pushed in from Service Workers. The change in HTTP-network fetch takes care of requests pushed in from the network (either inside or outside of a Service Worker). Where else do you think we ought to be setting the CSP list?